include/tunnler/tunnler.h

```diff
@@ -15,6 +16,8 @@ using MacAddress = std::array<uint8_t, 6>;
 /// A special MAC address that tells the room we're joining to assign us a MAC address automatically.
 const MacAddress NoPreferredMac = { 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF };
 
+const std::chrono::milliseconds sleep_time(1);
```

Comment:
Add documentation ///< Time the Room::ServerLoop() and RoomMember::ReceiveLoop() sleeps during echo loop